keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ArtifactController.kt

```diff
@@ -38,31 +40,21 @@ class ArtifactController(
   fun submitArtifact(@RequestBody echoArtifactEvent: EchoArtifactEvent) {
     echoArtifactEvent.payload.artifacts.forEach { artifact ->
       if (artifact.type.equals(deb.toString(), true) && artifact.isFromArtifactEvent()) {
-        publisher.publishEvent(ArtifactEvent(listOf(artifact), emptyMap()))
+        debianArtifactSupplier.publishArtifact(ArtifactEvent(listOf(artifact), emptyMap()))
       } else if (artifact.type.equals(docker.toString(), true)) {
-        publisher.publishEvent(ArtifactEvent(listOf(artifact), emptyMap()))
+        dockerArtifactSupplier.publishArtifact(ArtifactEvent(listOf(artifact), emptyMap()))
       } else {
         log.debug("Ignoring artifact event with type {}: {}", artifact.type, artifact)
       }
     }
   }
 
-  @PostMapping(
-    path = ["/register"],
```

Comment:
Previously used to register artifacts before we had delivery configs. Artifacts are now registered via delivery configs.